app/services/questionnaire_package_import_service.rb

```
@@ -0,0 +1,556 @@
+      item.questionnaire = questionnaire
+
+      imported_seq = item.seq
+      item.save!
+      item.update_column(:seq, imported_seq) if imported_seq.present? && item.seq.to_s != imported_seq.to_s
+
+      imported_items[item_source_key(row['questionnaire_name'], row['questionnaire_instructor_name'], row['seq'], row['txt'])] = item
+      imported_counts[:items] += 1
+    end
+
+    imported_items
+  end
```

RuboCop (1.86.1)

[convention] Assignment Branch Condition size for `import_items` is too high. [<9, 35, 6> 36.63/20]

(Metrics/AbcSize)